Makefile

```diff
@@ -231,6 +231,7 @@ coverage: .init
 	$(DOCKER_CMD) contrib/hack/coverage.sh --html "$(COVERAGE)" \
 	  $(addprefix ./,$(TEST_DIRS))
 
+.PHONY: test test-unit test-integration test-e2e
```

Comment:
I think this is supposed to be "test-dep" instead of "test-e2e"

Reply:
test-dep is already called out as phony on line 384 (correctly now with Moran's fix).  I think it's correct to add test-e2e as well.